integration-tests/goss/goss-shared.yaml

```diff
@@ -177,3 +177,6 @@ matching:
         - have-key-with-value:
             foo: bar
         - have-key: baz
+env-var:
+  GOSS_TEST: true
```

Comment:
I think this needs to be `GOSS_TEST: "true"` to ensure "true" is a string not a boolean value.